android/src/main/java/io/mosip/tuvali/common/events/Event.kt

```diff
@@ -0,0 +1,7 @@
+package io.mosip.tuvali.common.events
+
+interface Event {
+  val type: String
+  val args: HashMap<String, String>?
```

Comment:
this field can be renamed to `details` instead of `args`. Once it is serialised the structure would look like this `{"type":"onError": "details": {"message": "", "code": ""}}`

Reply:
On second thoughts, `args` field can be completely avoided and let the concrete type implementing the interface decide its own structure.

Reply:
outdated. please review again